contracts/Super1155.sol

```diff
@@ -639,6 +598,51 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
       _amounts, _data);
   }
 
+
+  /**
+    Transfer on behalf of a caller or one of their authorized token managers
+    items from one address to another.
+
+    @param _from The address to transfer tokens from.
+    @param _to The address to transfer tokens to.
+    @param _id The specific token ID to transfer.
+    @param _amount The amount of the specific `_id` to transfer.
+    @param _data Additional call data to send with this transfer.
+  */
+  function safeTransferFrom(address _from, address _to, uint256 _id,
+    uint256 _amount, bytes calldata _data) external override virtual {
+      safeBatchTransferFrom(_from, _to, _asSingletonArray(_id), _asSingletonArray(_amount), _data);
+    /*
```

Comment:
If this works please remove the dead comments.

Reply:
Ok